taxonomies/categories.txt

```diff
@@ -42896,7 +42896,7 @@ ciqual_food_name:fr:Yaourt, lait fermenté ou spécialité laitière, nature, su
 
 <fr:Spécialités laitières
 <en:Plain fermented dairy desserts
-fr:Spécialités laitières nature non sucrées
+fr:Spécialités laitières nature non sucrées, Spécialité laitière nature
```

Comment:
shouldn't "Spécialité laitière nature" be the main name in fr?

Reply:
Yes it could, I just want it to be explicite that is it without sugar. In Spain you can have "natural" yogurts that are sweetened, but in France I don't think it is the case. I changed it